<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginDialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>account/login.ico</normaloff>account/login.ico</iconset>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/account/account/login.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwdLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
